nucleus/dataset.py

```diff
@@ -389,6 +461,7 @@ def append(
               3D data to drastically increase throughput. Default is False.
 
         Returns:
+            For scenes:
```

Comment:
Uh oh, [Sphinx is assuming this is a return type](https://scale-ai-nucleus-python-sdk--153.com.readthedocs.build/en/153/api/nucleus/index.html#nucleus.Dataset.append) (think it does this for any text on the first line preceding a colon)...

Just tested locally, this will format things better (also prefer generic "payload" to `UploadResponse` since we're not exposing this to users iirc):
```
Returns:
    For scenes
        If synchronous, returns a payload describing the upload result::
                {
                    "dataset_id: str,
                    "new_items": int,
                    "updated_items": int,
                    "ignored_items": int,
                    "upload_errors": int
                }
        Otherwise, returns an :class:`AsyncJob` object.

    For images
        If synchronous returns payload. Otherwise :class:`AsyncJob`.
```

Reply:
Fixed!